community/lld/APKBUILD

```diff
@@ -2,13 +2,13 @@
 # Contributor: Jakub Jirutka <jakub@jirutka.cz>
 # Maintainer: Jakub Jirutka <jakub@jirutka.cz>
 pkgname=lld
-pkgver=8.0.1
+pkgver=9.0.0
 pkgrel=0
 _llvmver=${pkgver%%.*}
 pkgdesc="The LLVM Linker"
 url="https://llvm.org"
```

Comment:
```suggestion
url="https://llvm.org/"
```

Reply:
Does the trailing slash really make a difference?

Reply:
Not really, it's just to normalize